plugins/mobile-plugin/src/test/java/pagefactory/mobile/pages/main/MainPage.java

```diff
@@ -0,0 +1,23 @@
+package pagefactory.mobile.pages.main;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import ru.sbtqa.tag.pagefactory.mobile.utils.SwipeUtils;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'ru.sbtqa.tag.pagefactory.mobile.utils.SwipeUtils'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)